legal-api/docker-compose.yml

```diff
@@ -4,10 +4,9 @@ services:
     image: "postgres:11"
     container_name: "legal_api_postgres"
     ports:
-      - "5432:5432"
+      - "${DATABASE_TEST_PORT}:5432"
     environment:
-      - LC_ALL=C.UTF-8
-      - POSTGRES_DB=legal_test
-      - POSTGRES_USER=legal
-      - POSTGRES_PASSWORD=
-      - POSTGRES_PORT=5432
+      LC_ALL: "C.UTF-8"
+      POSTGRES_DB: "${DATABASE_TEST_NAME}"
+      POSTGRES_USER: "${DATABASE_TEST_USERNAME}"
+      POSTGRES_PASSWORD: "${DATABASE_TEST_PASSWORD}"
```

Comment:
Using this opportunity to clean up our tooling.  This change is unrelated to the ticket.  I can remove the file if anyone objects to its inclusion.